mlrun/projects/project.py

```diff
@@ -3773,6 +3774,62 @@ def delete_api_gateway(
 
         mlrun.db.get_run_db().delete_api_gateway(name=name, project=self.name)
 
+    def store_alert_config(self, alert_data: AlertConfig, alert_name=None):
+        """
+        Create/modify an alert.
+        :param alert_data: the data of the alert.
+        :param alert_name: name of the alert.
+        """
+        db = mlrun.db.get_run_db(secrets=self._secrets)
+        if alert_name is None:
+            alert_name = alert_data.name
+        return db.store_alert_config(alert_name, alert_data.dict(), self.metadata.name)
+
+    def get_alert_config(self, alert_name):
```

Comment:
```suggestion
    def get_alert_config(self, alert_name) -> AlertConfig:
```
This is missing a hint as well as a :return: docstring notation.
Since the project object is client-facing, it's usually "talking" objects, so should return the actual `AlertConfig` (see comment on httpdb for returning the actual object).